src/components/select-field/BaseSelectField.js

```diff
@@ -221,12 +223,17 @@ class BaseSelectField extends React.Component<Props, State> {
 
     handleBlur = (event?: SyntheticFocusEvent<>) => {
         const { isOpen } = this.state;
+        const { blurExceptionClassNames = [] } = this.props;
+
+        const exceptionClasses = ['search-input', 'select-button', ...blurExceptionClassNames];
+
         if (
             isOpen &&
             event &&
             event.relatedTarget &&
-            !(event.relatedTarget: window.HTMLInputElement).classList.contains('search-input') &&
-            !(event.relatedTarget: window.HTMLInputElement).classList.contains('select-button')
+            exceptionClasses.every(
+                className => event && !(event.relatedTarget: window.HTMLInputElement).classList.contains(className),
```

Comment:
minor: do you need the `event &&` check, since thats already checked on line 232?

Reply:
we don't but Flow was complaining that `event` could be empty. I think it's because of the scoping?